drivers/health_flexible_service/app/controllers/health_flexible_service/warehouse_reports/member_expiration_controller.rb

```diff
@@ -0,0 +1,30 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HealthFlexibleService::WarehouseReports
+  class MemberExpirationController < ApplicationController
+    include WarehouseReportAuthorization
+    before_action :require_can_administer_health!
```

Comment:
@awisema are we sure this is the correct permission?

Reply:
That sounds better to me.  @gigxz , can you adjust both of those?

Reply:
Yep, done